src/resources/commandTemplates/templates.ActionCommand.ts

```diff
@@ -399,6 +411,36 @@ export default class ActionCommand extends ResponsiveSlashCommandSubcommandBuild
         },
         { sendNoticeFirst: true, emoji: ':hammer:', pastTense: 'banned' },
       ],
+      [
+        {
+          name: 'Add Mature',
+          value: 'add_mature',
+        },
+        async (member) => {
+          return member.manageable;
+        },
+        async (member, reason) => {
+          if (!member.manageable) return false;
+          const SNOWFLAKE_MAP = await getSnowflakeMap();
+          return !!(await member.roles.add(SNOWFLAKE_MAP.Mature_Roles, reason));
+        },
+        { emoji: ':white_check_mark:', pastTense: 'added to the mature role' },
```

Comment:
not sure where 'added to the mature role', 'removed from the mature role' are used but i'd rephrase the grammar

Reply:
Resolved!